packages/babel-plugin-proposal-pipeline-operator/src/index.js

```diff
@@ -1,64 +1,23 @@
 import { declare } from "@babel/helper-plugin-utils";
 import syntaxPipelineOperator from "@babel/plugin-syntax-pipeline-operator";
-import { types as t } from "@babel/core";
+import minimalVisitor from "./minimalVisitor";
+import smartVisitor from "./smartVisitor";
 
-export default declare(api => {
+export default declare((api, options) => {
   api.assertVersion(7);
 
+  // FIXME: Hacky... Need a way to get rid of this
+  delete smartVisitor._exploded;
+  delete smartVisitor._verified;
```

Comment:
I did this because the plugin validation kept complaining that `.visitor["_exploded"]` and `.visitor["_verified"]` must be functions. Is there other way to honor that?

Reply:
I haven't dug the root cause yet, but it seems like they are inserted by the plugin execution infrastructure. They somehow only showed up after the visitors were extracted to their own modules. When the visitors were declared inside the `declare` block itself, this doesn't arise. Maybe the visitor is getting reused between plugin executions?

Reply:
This was fixed by 36124fe because it eliminates the subtraversal. Thanks for the tip, @nicolo-ribaudo!